ansible/roles/kstest-master/files/scripts/kstests_history.py

```diff
@@ -53,13 +53,13 @@ def get_anconda_ver(result_path):
 count = 0
 old_isomd5 = ""
 for result_dir in sorted(os.listdir(results_path)):
-    count = count + 1
     result_path = os.path.join(results_path, result_dir)
 
     report_file = os.path.join(result_path, report_filename)
     if not os.path.exists(report_file):
         print("No {} found, skipping".format(report_file), file=sys.stderr)
         continue
+    count = count + 1
```

Comment:
Please move this one line below. It's easier to confuse it with the if above without an empty space.

I mean change this:
```
    continue
count = count + 1

anaconda_ver = ""

with open....
```

to:

```
    continue

count = count + 1
anaconda_ver = ""

with open...
```

Reply:
Fixed